Makefile

```diff
@@ -48,7 +48,7 @@ bench: ## Run benchmarks
 	pipenv run pytest benchmarks
 
 .PHONY: bench-compare
-bench-compare: nightly dev-packages install ## Run benchmarks and compare results with other JSON encoders
+bench-compare: ## Run benchmarks and compare results with other JSON encoders
```

Comment:
Would it hurt to add the dependencies `nightly dev-packages install` back? This way, we could also run `bench-compare` locally. If it interferes with the pipeline in an unpleasant way I'd be fine either way.